ROMFS/px4fmu_common/init.d-posix/airframes/10043_sihsim_standard_vtol

```diff
@@ -0,0 +1,116 @@
+#!/bin/sh
+#
+# @name SIH Standard VTOL
+#
+# @type Simulation
+# @class VTOL
+#
+# @output Motor1 MC motor front right
+# @output Motor2 MC motor back left
+# @output Motor3 MC motor front left
+# @output Motor4 MC motor back right
+# @output Motor5 Forward thrust motor
+# @output Servo1 Aileron
+# @output Servo2 Elevator
+# @output Servo3 Rudder
+#
+# @board px4_fmu-v2 exclude
+#
+
+. ${R}etc/init.d/rc.vtol_defaults
+
+# SIH in SITL changes. could we source these from a common file?  {{{
+# inspired by: diff ROMFS/px4fmu_common/init.d/airframes/4001_quad_x ROMFS/px4fmu_common/init.d-posix/airframes/10040_sihsim_quadx
+
+PX4_SIMULATOR=${PX4_SIMULATOR:=sihsim}
+PX4_SIM_MODEL=${PX4_SIM_MODEL:=standard_vtol}
+
+param set-default SENS_EN_GPSSIM 1
+param set-default SENS_EN_BAROSIM 1
+param set-default SENS_EN_MAGSIM 1
+
+param set-default EKF2_GPS_DELAY 0
+
+# battery check disabled below already
+# HIL_ACT_FUNC* set below -- do we need PWM_MAIN_FUNC* as well?
+# SIH_VEHICLE_TYPE set below too.
+
+# }}}
+
+param set-default VT_B_TRANS_DUR 5
+param set-default VT_ELEV_MC_LOCK 0
```

Comment:
Also here, I would remove it and keep the simulation setup as close to default setup as possible.